tools/stability-checker/README.md

```diff
@@ -1,37 +1,54 @@
+Notifications with test executions summary are sent on a Slack channel.
 
 ## Installation
 
-You can install the Stability Checker on the Kyma cluster as a chart. Find the chart definition in the `deploy/chart` directory.
-
-> **NOTE:** You must install the chart after running the core tests, to avoid running the same tests in parallel.
-
-## Usage
+You can install the Stability Checker on the Kyma cluster as a helm chart. Find the chart definition in the `deploy/chart` directory.
+1. Configure kubectl and helm to point your cluster by specifying **KUBECONFIG** environment variable. 
+```
+export KUBECONFIG="/path/to/kubeconfig"
+```
+1. Provision volume with a testing script, which then will be used by the Stability Checker. For that purpose, you can use 
+script `local/provision_volume.sh`. The script copies all files placed in `local/input` directory to PV.
 
-Stability Checker does not contain testing scripts. The chart value `.Values.pathToTestingScript` defines which script the system runs.
-Ensure you have the file available in a persistent volume defined as `.Values.storage.claimName`, which is mounted as a `data` directory in the Stability Checker Pod.
+2. Install stability checker as a helm chart and provide proper configuration:
 
-To simulate the process of providing scripts, see the `local/provision_volume.sh` script, which populates the volume with files from the `local/input` directory.
+```
+helm install deploy/chart/stability-checker \
+  --set slackClientWebhookUrl="" \
+  ...
+  --namespace="kyma-system" \
+  --name="stability-checker"
 
-### Deliver the chart
+```
 
-Download the gzipped chart from:
+Below you can find configuration options:
 
-`https://github.com/kyma-project/stability-checker/raw/{branchName}/deploy/chart/stability-checker-0.1.0.tgz`
+ | Name | Default value | Description |
+ |------|---------------|-------------|
+storage.claimName |stability-test-scripts-pvc| Name of PVC which is attached to Stability Checker pod. Volume is visible in the pod under `/data` path. 
+pathToTestingScript |/data/input/testing.sh| Full path to the testing script. Because script is delivered inside PV, it has to start with `/data`.
+slackClientWebhookUrl |-| Slack client webhook URL.
+slackClientChannelId |-| Slack channel ID, starts with `#`.
+slackClientToken |-| Slack client token.
+testThrottle | 5m | Period between test executions. Purpose of this parameter is to give K8s time to clean up all resources after the previous test execution.
+testResultWindowTime | 6h | Notifications will be sent after this time and contains test executions summary for this period. 
+stats.enabled | false | If true, an output from test executions is analyzed to find statistics for every specific test. Detailed information about how many times every test failed and succeeded will be enclosed to the slack notification. Detecting test result is done by regular expressions defined in `stats.failingTestRegexp` and `stats.successfulTestRegexp`.
```

Comment:
``If set to `true`, an output from test executions is analyzed to find statistics for every specific test. Detailed information about the number of times every test failed and succeeded is enclosed to the Slack notification. Regular expressions defined in **stats.failingTestRegexp** and **stats.successfulTestRegexp** detect test results.`` 

Also: Add a sentence:
``You can configure these two parameters only if **stats.enabled** is set to `true`.``